run-tests.sh

```diff
@@ -8,7 +8,7 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 pydocstyle invenio_admin tests docs && \
-isort -rc -c -df && \
+isort invenio_admin tests --recursive --check-only --diff
```

Comment:
Sorry to insist but `--recursive` will be deprecated at some point... See outputs when running `isort` latest:

```
isort invenio-admin tests --recursive --check-only
/Users/ppanero/.virtualenvs/invenio-admin/lib/python3.6/site-packages/isort/main.py:907: UserWarning: W0501: The following deprecated CLI flags were used and ignored: --recursive!
  "W0501: The following deprecated CLI flags were used and ignored: "
/Users/ppanero/.virtualenvs/invenio-admin/lib/python3.6/site-packages/isort/main.py:911: UserWarning: W0500: Please see the 5.0.0 Upgrade guide: https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/
  "W0500: Please see the 5.0.0 Upgrade guide: "
```